### ISSUE TITLE
0.0.12-preview.23: non-existent assembly 'Unity.PerformanceTesting'

### ISSUE THREAD

**Issue by daschatten** [Unity-Technologies/EntityComponentSystemSamples#95]
Error:

`Assembly has reference to non-existent assembly 'Unity.PerformanceTesting' (Packages/com.unity.entities/Unity.Entities.PerformanceTests/Unity.Entities.PerformanceTests.asmdef)`

Fix:

Remove "Unity.PerformanceTesting" from Unity.Entities.PerformanceTests.asmdef

**Comment by devEyesgood** [comment]
I am having this same issue.  I cannot work with ECS at all.

**Comment by joshpowlison** [comment]
Ran into the same bug, thanks! @devEyesgood I replaced my manifest.json with this Repo's and it works now as well.

I'll reference this error if I find this causing any other problems down the road.

**Comment by sjk07** [comment]
yeah the `com.unity.test-framework.performance` package is never added as a dependency from the package manager, this is causing the issue...

There are install instructions here if you want to know a little more about how the dependencies are working:
https://docs.unity3d.com/Packages/com.unity.test-framework.performance@0.1/manual/index.html

**Comment by aeldron** [comment]
For some reason, the Performance testing API package was not showing on the advanced tab of the Package Manager.  I had to manually add it to my manifest.json:
    "com.unity.test-framework.performance": "0.1.49-preview"
and then it appeared in the Package Manager with an option to update it to 0.1.50-preview

**Comment by VladislavKostin** [comment]
Do we need Unity.PerformanceTesting? What does it contain? I just removed the reference as @daschatten suggested.

**Comment by aoighost** [comment]
@VladislavKostin without it build breaks, so I'd fix that.

**Comment by edgarhsanchez** [comment]
> For some reason, the Performance testing API package was not showing on the advanced tab of the Package Manager. I had to manually add it to my manifest.json:
> "com.unity.test-framework.performance": "0.1.49-preview"
> and then it appeared in the Package Manager with an option to update it to 0.1.50-preview

probably should be visible in the package manager without having to muck around with the manifest file right?

**Comment by rustinlee** [comment]
Just ran into this as well, adding `"com.unity.test-framework.performance": "0.1.49-preview"` made the package visible and resolved the error, but I don't get why the issue was closed when this still requires manual fixing from the end-developer

**Comment by equinox80** [comment]
> Just ran into this as well, adding `"com.unity.test-framework.performance": "0.1.49-preview"` made the package visible and resolved the error, but I don't get why the issue was closed when this still requires manual fixing from the end-developer

They probably closed it because they know the issue and have fixed it on newer builds. Maybe.

**Comment by JCaraballo113** [comment]
@equinox80 as of 2018.3.5f1 this still exists. Not sure why it was closed. Maybe a dupe?

**Comment by 2bam-at-tlon** [comment]
Still happens when updating projects from 2018.3 to 2019.1 and 2019.2.
Thanks for sharing this.
Adding manually `"com.unity.test-framework.performance": "0.1.49-preview"` to the manifest.json worked.
